ebuild-maintenance/git/text.xml

```diff
@@ -286,11 +275,19 @@ as --force) for the commit.</li>
 </ul>
 
 <p>
-Additionally, some developers prefer referencing bugs on the summary
-line using the <c>#nnnnnn</c> form. Developers are free to use either
-form, or both simultaneously to combine their advantages.
+Additionally, some developers prefer referencing Gentoo Bugzilla bugs on
+the summary line using the <c>#BUG-ID</c> form. Developers are free to use
```

Comment:
IMHO the `#nnnnnn` made it immediately clear that a numerical ID is meant, whereas one may wonder what a "BUG-ID" is. Please leave `#nnnnnn` in place.

Reply:
I've updated the _#BUG-ID_ references to _#nnnnnn_.